test/python/test_manif.py

```diff
@@ -224,6 +224,32 @@ def test_smallAdj(self, LieGroup, Tangent):
             delta.hat() @ delta_other.hat() - delta_other.hat() @ delta.hat()
         )
 
+    def test_Bracket(self, LieGroup, Tangent):
+        a = Tangent.Random()
+        b = Tangent.Random()
+
+        assert a.bracket(b) == Tangent.Bracket(a, b)
+
+        assert np.allclose(
+            Tangent.Bracket(a, b).hat(), (a.hat() @ b.hat()) - (b.hat() @ a.hat())
+        )
+
+        c = Tangent.Random()
+
+        assert (
+            Tangent.Bracket(a, Tangent.Bracket(b, c)) +
+            Tangent.Bracket(b, Tangent.Bracket(c, a)) +
+            Tangent.Bracket(c, Tangent.Bracket(a, b))
+        ).isApprox(Tangent.Zero())
+
+        assert Tangent.Bracket(a, b).isApprox(-Tangent.Bracket(b, a))
+
+    def test_Vee(self, LieGroup, Tangent):
+        a = Tangent.Random()
+
+        # assert a.isApprox(Tangent.Vee(a.hat()))
```

Comment:
Why is this assertion commented out? It seems to work correctly on my side.
* Creates a new tangent from static method Vee() which takes a.hat() as input.
* Compares it to a